configs/Kconfig

```diff
@@ -55,6 +55,10 @@ config CURSOR
     default n
     depends on !BACKEND_VNC
 
+config DROP_SHADOW
+    bool "Render drop shadow for active window"
+    default n
```

Comment:
Let's enable this feature by default, allowing developers to focus on further improvements.